arangodb-net-standard/TransactionApi/TransactionApiClient.cs

```diff
@@ -63,6 +76,146 @@ public TransactionApiClient(IApiClientTransport client, IApiClientSerialization
                 {
                     return DeserializeJsonFromStream<PostTransactionResponse<T>>(stream);
                 }
+
+                var error = DeserializeJsonFromStream<ApiErrorResponse>(stream);
+                throw new ApiErrorException(error);
+            }
+        }
+
+        /// <summary>
+        /// Abort a stream transaction by DELETE.
+        /// </summary>
+        /// /// <remarks>
+        /// https://www.arangodb.com/docs/3.6/http/transaction-stream-transaction.html#abort-transaction.
+        /// </remarks>
+        /// <typeparam name="TStreamTransactionResult">Type to use for deserializing the object returned by the transaction function.</typeparam>
+        /// <param name="transactionId">The transaction identifier.</param>
+        /// <exception cref="ApiErrorException">
+        /// With ErrorNum 1653 if the transaction cannot be aborted.
+        /// With ErrorNum 10 if the transaction is not found.
+        /// </exception>
+        /// <returns>Response from ArangoDB after aborting a transaction.</returns>
+        public virtual async Task<PostTransactionResponse<TStreamTransactionResult>>
```

Comment:
I think we have better use a separate class instead of `PostTransactionResponse`. Using a generic parameter is uncessary here. It also means the consumer will have to figure out what to pass, but this is a standard ArangoDB response. I would recommand to create a specific class, e.g. `StreamTransactionResponse`.